lib/babe/babe.go

```diff
@@ -56,9 +56,6 @@ type Service struct {
 	// BABE authority keypair
 	keypair *sr25519.Keypair // TODO: change to BABE keystore
 
-	// Current runtime
-	rt runtime.Instance
```

Comment:
Also remove `Runtime` from `ServiceConfig` struct.

Reply:
The test is using this runtime. I have created another issue to remove the runtime. #1674